test/runner/e2e_test.cpp

```diff
@@ -195,13 +196,23 @@ int main(int argc, char** argv) {
     try {
         // Main logic
         std::string test_dir;
+        std::string import_data_dir;
         char* env_test_dir = std::getenv("E2E_TEST_FILES_DIRECTORY");
+        char* env_import_data_dir = std::getenv("E2E_IMPORT_TEST_DIR");
         if (env_test_dir != nullptr) {
             test_dir = env_test_dir;
         } else {
             test_dir = "test/test_files";
         }
+
+        if (env_import_data_dir != nullptr) {
+            import_data_dir = env_import_data_dir;
```

Comment:
There should be a check that `env_import_data_dir` ends in `/`.

Even better would to test that directory exists.